react-native/react/native/listen-log-ui.js

```diff
@@ -1,10 +1,13 @@
+/* @flow */
 import engine from '../engine'
-import {logUiLog} from '../actions/notifications'
+import {log} from './log/logui'
+
+import type {Text as KBText, LogLevel} from '../constants/types/flow-types'
 
 export default function ListenLogUi () {
   engine.listenOnConnect('ListenLogUi', () => {
-    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}) => {
-      logUiLog(params)
+    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: KBText, level: LogLevel}, response: any) => {
+      log(params, response)
```

Comment:
Maybe the response should be here where things know about the rpc vs the helper below